freqtrade/optimize/backtesting.py

```diff
@@ -204,6 +204,12 @@ def load_bt_data(self) -> Tuple[Dict[str, DataFrame], TimeRange]:
         """
         self.progress.init_step(BacktestState.DATALOAD, 1)
 
+        if self.config['freqaimodel']:
+            self.required_startup += int((self.config['freqai']['train_period']*86400) /
```

Comment:
if you assume certain attributes to be in the config, they must be properly specified in constants so schema validation takes place.

otherwise, all options can be optional, and you can't trust anything to be there.
(i'd go the validated route though - otherwise it can be tough)